05-classification.Rmd

```diff
@@ -359,24 +369,29 @@ next chapter.
 
 **Distance between points** 
 
-How do we decide which points are the $K$ "nearest" to our new observation? We
-can compute the distance between any pair of points using the following
+We decide which points are the $K$ "nearest" to our new observation
+using the *straight-line distance* (we will often just refer to this as *distance*). 
+The straight-line distance between any pair of points $(x_a, y_a)$ 
+and $(x_b, y_b)$ on the x-y plane can be computed using the following
 formula: 
 
 $$\mathrm{Distance} = \sqrt{(x_a -x_b)^2 + (y_a - y_b)^2}$$
 ```{r 05-multiknn-0, echo = FALSE}
 new_point <- c(0, 3.5)
 ```
 
-> This formula -- sometimes called the *Euclidean distance* -- is simply the straight line distance between two points on the x-y plane with coordinates $(x_a, y_a)$ and $(x_b, y_b)$.
-
-Suppose we want to classify a new observation with perimeter of `r new_point[1]` and 
-concavity of `r new_point[2]`. Let's calculate the distances
+To find the $K$ nearest neighbours to our new observation, we compute the distance
+to each observation in our training data, and take the $K$ observations corresponding to the
+$K$ *smallest* distance values. For example, suppose we want to classify a new 
+observation with perimeter of `r new_point[1]` and 
+concavity of `r new_point[2]`, shown in \@ref(fig:05-multiknn-1). Let's calculate the distances
 between our new point and each of the observations in the training set to find
-the $K=5$ observations in the training data that are nearest to our new point. 
+the $K=5$ training observations that are nearest to our new point. 
```

Comment:
Can we simply say 5 nearest neighbours here?
```suggestion
the 5 nearest neighbours that are closest to our new point. 
```

Reply:
edited